### ISSUE TITLE
Feature request: email notification when *all* reviews are in

### ISSUE THREAD

**Issue by smilius42** [CCSDForge/episciences#123]
Dear colleagues,
it might be a useful feature if editors get a notification when *all* reviews for a submission have been received. This should *not* be a separate email but be sent as one email with the notification for the last review (to minimize email traffic for editors).
So perhaps a new automatic email 'Last rating completed' should be implemented in addition to the already existing 'Completed rating' email template.
BTW, 'rating' should be 'review'.

Best,
Stefan

p.s. Copied from https://support.ccsd.cnrs.fr/SelfService/Display.html?id=115679

**Comment by a3nm** [comment]
I agree with the proposal and would like to point out that "rating" as an unconventional name for "review" has also caused some confusion in our Episciences journal. It'd be great if it could be changed.

**Comment by smilius42** [comment]
Update: at our Episciences journal we just had a case of an editor not following up on a submission because he did not keep track how many reviews were still missing, and so he did not act after the last review was received for more than half a year. 

I would therefore like to add to this feature request the following: 

In addition to the email notifying the editor when all reviews have been received there should be an automatic reminder email which is sent to the editor after some time if he does not react by sending the authors a notification.

**Comment by rtournoy** [comment]
> I agree with the proposal and would like to point out that "rating" as an unconventional name for "review" has also caused some confusion in our Episciences journal. It'd be great if it could be changed.

I think we have changed for review in several places,let us know if we forgot something

**Comment by rtournoy** [comment]
> Update: at our Episciences journal we just had a case of an editor not following up on a submission because he did not keep track how many reviews were still missing, and so he did not act after the last review was received for more than half a year.
> 
> I would therefore like to add to this feature request the following:
> 
> In addition to the email notifying the editor when all reviews have been received there should be an automatic reminder email which is sent to the editor after some time if he does not react by sending the authors a notification.

Dear @smilius42, @a3nm,
Do you think it will be OK if we create an automatic reminder feature for this case? 
thanks

**Comment by a3nm** [comment]
Yes I think it would be really useful to have a different email template when the last review arrives, given that this requires the editor to take action. Currently I must do this manually by checking when a review is submitted whether it is the last one or not, and ping the editor if it is.
